packages/hasher/src/__tests__/hashOfFiles.test.ts

```diff
@@ -10,25 +9,18 @@ const { createDefaultConfig } = jest.requireActual("backfill-config");
 jest.mock("backfill-config");
 
 const mockedDependency = <jest.Mock<Config>>createConfig;
```

Comment:
I think we can remove these since we're passing the globs directly, instead of relying on createConfig.